src/fastlane/release_actions/lib/fastlane/plugin/release_actions/helper/commit/commits.rb

```diff
@@ -37,7 +39,4 @@ def empty?
     commits.empty?
   end
 
-  private
-
-  attr_reader :commits
```

Comment:
For Flutter we need to write changelogs for each package, which requires looking at each commit to group them by their scopes. Making the commits reader public seems to be the simplest way to achieve this (moved the `attr_reader` declaration to the top, line 4). Does this sound okay?